<launch>
  <arg name="rgb_camera_info" value="/wav_camera_info"/>
  <arg name="rgb_img_rect" value="/wav_image_color"/>  <!--Rectified color image-->
  <arg name="depReg_imgrect" value="/wav_depth_image"/>  <!--Rectified depth image-->
  <arg name="out_cloud" value="wav_pcl"/>

  <node pkg="nodelet" type="nodelet" name="standalone_nodelet" args="manager" output="screen"/>

  <!-- Construct point cloud of the rgb and depth topics -->
  <node pkg="nodelet" type="nodelet" name="points_xyzrgb" args="load depth_image_proc/point_cloud_xyzrgb standalone_nodelet --no-bond">
    <remap from="rgb/camera_info" to="$(arg rgb_camera_info)" />  
    <remap from="rgb/image_rect_color" to="$(arg rgb_img_rect)"/>  
    <remap from="depth_registered/image_rect" to="$(arg depReg_imgrect)"/>  
    <remap from="depth_registered/points" to="$(arg out_cloud)"/>
  </node>
</launch>